library/src/meta/numbering.rs

```diff
@@ -260,6 +260,13 @@ enum NumberingKind {
     Symbol,
     Hebrew,
     Chinese,
+    // TODO: Pick the numbering pattern based on languages choice.
+    // As the `1st` numbering character of Chinese (Simplifed) and
+    // Chinese (Traditional) is same, we are unable to determine
+    // if the context is Simplified or Traditional by only this
+    // character.
+    #[allow(unused)]
+    TradChinese,
```

Comment:
This should be `ChineseSimplified` and `ChineseTraditional` as well, to make it consistent with the other enum.